### PR TITLE
Debug3

### PR DESCRIPTION
**Pull request by adahuangsx**

### CODE REVIEW

**Comment by adahuangsx** [comment]
navigator merged to debug

File: generator.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#mysql.connector=C:/Users/cheke/.m2/repository/mysql/mysql-connector-java/5.1.47/mysql-connector-java-5.1.47.jar
+mysql.connector=C:/Users/Tianfang Ma/.m2/repository/mysql/mysql-connector-java/8.0.15/mysql-connector-java-8.0.15.jar
+mysql.url=jdbc:mysql://127.0.0.1:3306/upmc?serverTimezone=America/New_York&useSSl=false
+mysql.user=root	
+mysql.password=matian@VRF30805
```
Comment (CptTZ) [inline generator.properties]: Please change password right now

File: application.properties [hunk 2]
```diff
@@ -0,0 +1,29 @@
+spring.servlet.multipart.max-file-size=1024MB
+spring.servlet.multipart.max-request-size=1024MB
+
+spring.datasource.url=jdbc:mysql://127.0.0.1:3306/upmc?serverTimezone=America/New_York&useSSl=false
+spring.datasource.username=root
+spring.datasource.password=matian@VRF30805
```
Comment (CptTZ) [inline application.properties]: Please change password right now

File: generator.properties (hunk 1 above)
Comment (adahuangsx) [reply generator.properties]: This file is redundant after the merge #83 . I have deleted it. The original properties files are under "/resources" not root.

**Pull request merged**